sematic/ui/src/index.tsx

```diff
@@ -183,6 +185,49 @@ function Router() {
   return <RouterProvider router={router} />;
 }
 
+(async () => {
+  const currentUrl = new URL(window.location.toString());
+  const currentHostName = await sha1(currentUrl.hostname);
+
+  posthog.init( 
+    'phc_nJlFf7MpsrzF5pPaSEQi5GKyTSDjHRcqqL808VLRNXc', { 
+      api_host: 'https://app.posthog.com',
+      autocapture: false,
+      debug: true,
+      disable_session_recording: true,
+      capture_pageview: false,
+      capture_pageleave: false,
+      persistence: "localStorage",
+      property_blacklist: [
+        '$referrer', '$referring_domain', '$initial_current_url', '$initial_referrer',
+        '$initial_referring_domain'
+      ],
+      loaded: () => {
+        setupPostHogOptout();
+        posthog.capture('$pageview');
+      },
+
+      sanitize_properties: (properties: Properties, event_name: string) => {
+        const currentUrl = new URL(window.location.toString());
+
+        // Use obfuscated host name
+        currentUrl.hostname = currentHostName;
+
+        if ('$current_url' in properties) {
+          properties['$current_url'] = currentUrl.toString();
+        }
+
+        if ('$host' in properties) {
+          properties['$host'] = currentUrl.host;
+        }
+
+        return properties;
```

Comment:
Can we capture the version of Sematic they're using as well?

Reply:
Yes, and `process.env.NODE_ENV`.

Although for version, I'm not sure the front-end is aware of it at this time. We would have to return it as part of the `/env` endpoint. This would also be useful to display in the app the current version, and also the current git sha in dev mode so that we can quickly confirm when we deploy to dev namespaces.

Reply:
there is an endpoint already serving it up: https://dev1.dev-usw2-sematic0.sematic.cloud/api/v1/meta/versions

Reply:
Front-end can acquire an environment variable at compile time. So we can read the version from the Python side and then feed it to the ts compiler. A plain text file holding the version will make this the easiest.  Git sha can be done the same way. 

But calling an endpoint to get the server version is better. Because that's what the FE really connects to (considering the extreme cases that the FE doesn't connect to the server it is built with)

I will add those 3 (NODE_ENV, GIT_HASH, SERVER_VERSION)